### PR TITLE
Problem: Banning validator pubkey is not necessary

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- Remove banning of double signed validator key.

### CODE REVIEW

File: docs/modules/staking-state.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ There are several variants of it:
 The only way to transit to active validator is by executing `NodeJoinTx`, the preconditions are:
 
 - `bonded >= min_required_staking`
-- The validator pubkey/address is not banned or already used
+- The validator pubkey/address is not already used
```
Comment (devashishdxt) [inline docs/modules/staking-state.md]: If a validator manually unbonds and wishes to rejoin later, will they need a new pubkey/address?

Comment (yihuang) [reply docs/modules/staking-state.md]: > If a validator manually unbonds and wishes to rejoin later, will they need a new pubkey/address?

It's ok to re-use your own keys, I just added a sentence to make it clear.

**Review by devashishdxt — COMMENTED** [review]
Is there any reason for this change?

**Comment by yihuang** [comment]
> Is there any reason for this change?

Because after discussing with @tomtau , we think do the banning in chain core don't help much, if our purpose is only to prevent operator making mistakes (like reusing a leaked key), we can do it in client.

**Comment by tomtau** [comment]
> Is there any reason for this change?

making it simple -- some of blacklist use cases are not relevant; others are handled by that old key list: https://github.com/crypto-com/chain-docs/blob/master/docs/modules/staking-state.md#inactive-validator-re-join-with-different-validator-key

**Pull request merged**